### DISCUSSION TITLE
Install two mmdetection separately

### DISCUSSION THREAD

**Discussion by Jacobew** [open-mmlab/mmdetection#9081] [Q&A] [answered]
Hi, is it possible to install two separate mmdetection in Linux? I'm developing two features at the same time and checkout git branch frequently seems a little bit tedious for me.

**Comment by ZwwWayne** [comment, marked as answer]
You can git clone two reprositories and developed them in different repos.
By setting `PYTHONPATH` when running the code, you can run these two repos safely without affecting each other.
